samples/sampler/stories/default/DayPicker.js

```diff
@@ -15,13 +15,15 @@ export const _DayPicker = (args) => (
 		<DayPicker
 			aria-label={args['aria-label']}
 			disabled={args['disabled']}
+			selected={args['selected']}
 			onSelect={action('onSelect')}
```

Comment:
Normally, we place props in alphabetical order.
```suggestion
			onSelect={action('onSelect')}
			selected={args['selected']}
```

Reply:
I will remove the entire code change of sampler